profile/README.md

```diff
@@ -18,3 +18,21 @@ You can learn more about this subproject on the [website](https://www.jenkins.io
 
 The GitHub organization of the main Jenkins community including repositories of the [Jenkins core](https://github.com/jenkinsci/jenkins), plugins, libraries and developer tools is here:
 https://github.com/jenkinsci/
+
+
+### Jenkins Contributor Awards 2023
+
+The annual Jenkins award  is happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
+
+Nomination categories are:
+- Most Valuable Jenkins Advocate → Nominate Someone
+- Most Valuable Jenkins Contributor → Nominate Someone
+- Jenkins Security MVP → Nominate Someone
+
+Award timeline:
```

Comment:
```suggestion
Awards timeline:
```

Reply:
I will ask Alyssa who is author of the text. This requires a native english speaker.